plugins/clickai-codex/skills/decide/advisory-board/scripts/_conductor/constants.py

```
@@ -0,0 +1,426 @@
+MODEL_PRICING_USD_PER_MTOK = {
+    # (input $/MTok, output $/MTok) — uncached list rates; cache reads are cheaper
+    "claude-fable-5": (10.00, 50.00),          # Anthropic
+    "gpt-5.5": (5.00, 30.00),                  # OpenAI
+    "gemini-3.5-flash": (1.50, 9.00),          # Google
+    "grok-4.5": (2.00, 6.00),                  # xAI
+    "Gemini 3.5 Flash (High)": (1.50, 9.00),   # antigravity display name — same model family
+    "llama3.3": (0.00, 0.00),                  # local seat — no per-token cost at all
+}
+
+
+def price_band_usd(model: str, tokens_in: "Optional[int]" = None,
```

Ruff (0.16.3)

[error] Undefined name `Optional`

(F821)